README.md

```diff
@@ -101,6 +101,19 @@ All of the roles internally have the notion of the following "steps" that are co
 
 Running a playbook will run the respective roles for each of these. If instead you want to run a subset you can use the operation argument as such `ansible-playbook -e operation=run`.
 
+#### JFR Hyperfoil
+
+To record Hyperfoil into JFR files we need to enable the options. It is possible to record on the controller and the agents separately, which with a different approach.
+To collect on the controller, when running the controller benchmark to start, pass the `-e hyperfoil_jfr=true` argument.
```

Comment:
Are you sure this didn't work for the agent as well? I seem to remember that it used to generate a jfr on the agent, correct me if I am wrong though.

Reply:
Yeah, I couldn't get it to collect on the agents other than the java_args. The https://github.com/Hyperfoil/hyperfoil_test doesn't have an option for JFR and the deployment (https://github.com/Hyperfoil/Hyperfoil/blob/master/clustering/src/main/java/io/hyperfoil/deploy/ssh/SshDeployedAgent.java#L96) also doesn't pass the options from the controller to the agents.